### DISCUSSION TITLE
Virtual Chassis with LAGs

### DISCUSSION THREAD

**Discussion by bradbendy** [netbox-community/netbox#12153] [Q&A] [answered]
Hopefully I am missing something but how do we setup LAG/LACP bundles inside a VC? If I have a VC with two members and need a LAG to span across those two members I don't see how to do it. Only way I see possible is do two separate LAGs on each member but then you just have to know the LAG is really across those members.

Has anyone else ran into this or am I missing something?

Thanks!

**Comment by candlerb** [comment, marked as answer]
I think you must be missing something:
1. Create the VC
2. Go to the master device (which will show you the interfaces on all devices)
3. Create a LAG including whichever interfaces you like

Does that not work? If not, please show the exact steps you're doing.

Note also: before you create the VC, you should rename the interfaces (there's a bulk rename facility for this).  So if the device type has interfaces Gi1/0/1-Gi1/0/24, then on the second device rename all these interfaces to Gi2/0/1-Gi2/0/24, *before* joining it to the VC.

**Comment by bradbendy** [comment]
I was missing the master part, once I did that all the interfaces do show up. I figured it was something easy, thanks much!!
